synapse/tests/test_lib_cell.py

```diff
@@ -109,9 +109,9 @@ class CellTest(s_t_utils.SynTest):
                     await self.asyncraises(s_exc.AuthDeny, proxy.icando('foo', 'newp'))
 
                     # happy path perms
-                    await user.addRule((True, ('hive:set', 'foo', 'bar')))
-                    await user.addRule((True, ('hive:get', 'foo', 'bar')))
-                    await user.addRule((True, ('hive:pop', 'foo', 'bar')))
+                    await visi.addRule((True, ('hive:set', 'foo', 'bar')))
```

Comment:
These rules are OBE and the associated example is no bueno anymore

Reply:
Removing them breaks the test so i'm gonna call this a separate ticket since the only reason i've got deltas here is a variable name change.

Reply:
tracking this separately.